lib/cherry_silt/entity.rb

```diff
@@ -14,11 +14,39 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-class Entity
-	attr_accessor :name
-	attr_accessor :long_name
-
-	def initialize(name)
-		@name = name
-	end
-end
\ No newline at end of file
+##
+module CherrySilt
+  ##
+  class Entity
+    attr_accessor :name
+    attr_accessor :long_name
+
+    def initialize(name, long_name = nil)
+      @name = name
+      @long_name = long_name
```

Comment:
I don't know. it was there when you created the file in commit 030243b9a9cbed899e1584560321939eb6b2dd94